pkgs/unified_analytics/lib/src/enums.dart

```diff
@@ -56,6 +56,11 @@ enum DashEvent {
     description: 'Information for a flutter build invocation',
     toolOwner: DashTool.flutterTool,
   ),
+  flutterPubResult(
+    label: 'flutter_pub_result',
```

Comment:
not a blocker for this PR, but just wondering, do you know if long-term we would need this flutter tool event once pub has its own analytics via unified_analytics?

Reply:
Hmm that's a good question, and i would think it may still be useful to know which `pub` subcommands users are running within the flutter context? so yes we can track usage of `dart pub get` when we migrate pub, but is it also helpful to track `flutter pub get`?... if the answer is no, then this event likely won't be useful

From what I can see, context is just the lists from [here](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/dart/pub.dart#L67-L79)

And result is always either "success" or "failure"

Reply:
I think this is a question for the pub team--I don't know that anyone else uses these numbers.

Note, today we're intentionally suppressing pub's own analytics (https://github.com/christopherfujino/flutter/blob/disable-flutter-build-aar-plugins/packages/flutter_tools/lib/src/dart/pub.dart#L562), so flutter pub ONLY sends Flutter analytics, but I seem to recall that not being our intended final state.

cc @jonasfj @sigurdm

Reply:
We left the analytics out because we were unclear if the pdd covered sending dart analytics from the flutter tool.

I think with the unified analytics all should be good, and we can stop suppressing those, and potentially get actual useful data.

Reply:
Oh that's good to know, given that we now have the approvals, @christopherfujino are you in favor of deprecating this event? I think there may still be value in tracking successes/failures across different pub invocations from the flutter tool over time... while the pub team can collect more in depth analytics as well on their end.

Reply:
I am in favor of deleting it, but I will leave the final decision to the pub team, IDK if they use it or potentially would.

Reply:
Cool, I'll just close this PR for now and check offline. Will reopen if it's still necessary

Reply:
Just to loop back after conversation with @sigurdm and @jonasfj 

This event can remain deprecated since we are already collecting success/failure events from the `Event.flutterCommandResult` call site in `flutter_command.dart`